### PR TITLE
Add a doc about Module Self Configurator feature

### PR DESCRIPTION
**Pull request by thomasnares**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 8.x
| Description?  | Add docs regarding module Self Configurator feature

### CODE REVIEW

File: modules/configure-with-cli.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+```
+
+```yaml
+configuration:
+    delete:
+        - "PAYPAL_ONBOARDING"
+```
+
+{{% notice info %}}
+Please note that you can only `update` or `delete` values: `create` values are only done when installing the module.
```
Comment (PrestaEdit) [inline modules/configure-with-cli.md]: You can add value. The update will create if needed.

Comment (thomasnares) [inline modules/configure-with-cli.md]:
```suggestion
Please note that you can `create` values by specifying them in the `update` section.
```

**Review by PrestaEdit — APPROVED** [review]

File: modules/configure-with-cli.md (hunk 1 above)
Comment (thomasnares) [reply modules/configure-with-cli.md]: thanks, fixed 👍

**Review by matks — APPROVED** [review]
Looks great to me thanks @thomasnares

**Pull request merged**